
syntax="proto2";

package ProtoMsg;

//请求
message MemRegisterRequest{
	required string uId = 1;// 学生id、教师id
	required string memName = 2;// 学生姓名、教师姓名
	required string token = 3;//token
	required int32 memType = 4;//1：学生 2：教师
	required string classId = 5;//课堂Id（课表Id）
}

//返回
message MemberResponse{
	required int64 mid = 1;
	required string memName = 2;
	required string token = 3;
}

/*
 * 重连消息。
 * 		服务器通知用户重连，会接收到 MemberReconnect（action = "reconnection"），用户禁止操作，开始同步轨迹。
 * 		完成轨迹同步后，向服务端发送 MemberReconnect（action = "ready"），此时等待其他用户完成同步。
 * 		其他用户也完成同步后，会接收到 MemberReconnect（action = "success"），此时可以继续上课。
 */
